python/tank/deploy/tank_commands/copy_apps.py

```diff
@@ -50,7 +50,11 @@ def __init__(self):
 
     def run_noninteractive(self, log, parameters):
         """
-        API accessor
+        Tank command API accessor. 
```

Comment:
I love by the way that you took the time to document all of these. I'm really looking forward to a sphinxed documentation of core and see where are the holes. :)